framework/helpers/BaseHtml.php

```diff
@@ -430,11 +430,22 @@ public static function mailto($text, $email = null, $options = [])
      * the attributes of the resulting tag. The values will be HTML-encoded using [[encode()]].
      * If a value is null, the corresponding attribute will not be rendered.
      * See [[renderTagAttributes()]] for details on how attributes are being rendered.
+     * @since 2.0.12 It is possible to pass the "srcset" option as an array which keys are
+     * descriptors and values are URLs. All URLs will be processed by [[Url::to()]].
```

Comment:
Created issue for that: https://github.com/yiisoft/yii2-apidoc/issues/140
I think it is worth to have at some point, but it is not a priority right now.